cves/CVE-2014-0480.yml

```diff
@@ -144,8 +148,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: The vulnerability involved using two unescaped forward slashs (//), therefore, is something I think an automated tool could have discovered.
```

Comment:
There are a couple of minor grammatical errors here.

Reply:
Tweaked wording slightly.